src/test/common/clara/test_common.cljc

```diff
@@ -25,7 +25,8 @@
   []
   [:temperature [{temperature :temperature}] (< temperature 20) (= ?t temperature)])
 
-(defsession my-session [test-rule cold-query] :fact-type-fn :type)
+(defsession my-session [test-rule
+                        cold-query] :fact-type-fn :type)
```

Comment:
The changes here are because on my machine I have some weird compilation issues around this line that aren't consistently reproducible and don't show up in Travis, and I was experimenting to see if anything changed that.  I can probably remove these changes.